src/connectors/dispatcher/impl.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+const _ = require('lodash');
+const URI = require('urijs');
+const {host, insecure} = require('../../config').dispatcher;
+
+const Connector = require('../Connector');
+const metrics = require('../metrics');
+
+module.exports = new class extends Connector {
+    constructor () {
+        super(module);
+        this.postRunRequests = metrics.createConnectorMetric(this.getName(), 'postPlaybookRunRequests');
+    }
+
+    async postPlaybookRunRequests (dispatcherWorkRequest) {
+        const uri = new URI(host);
+        uri.segment('internal');
+        uri.segment('dispatch');
+
+        const result = await this.doHttp ({
+            uri: uri.toString(),
+            method: 'POST',
+            json: true,
+            rejectUnauthorized: !insecure,
+            headers: this.getForwardedHeaders(),
```

Comment:
You'll also need to build the `authorization` header. See https://github.com/RedHatInsights/playbook-dispatcher#internal-rest-interface for details

Reply:
@jharting I've the authorization header to the connector, however is the PSK for Remediations created in the playbook-dispatcher?  Or will it be created seperately?  I figured that it would be shared with Remediations as part of a secret.

Reply:
Yes, the key will be available in a secret. Remediations will need to read the key and send it with every request to playbook dispatcher.